### PR TITLE
Assessments/2022 12 08 control constructs

### PR DESCRIPTION
**Pull request by AWalnut1**
going literally insane

also the reason why i have so much already done on the first submit is it didnt sign me in for some reason, glad i caught it

### CODE REVIEW

**Comment by AWalnut1** [comment]
forgot to request your review, sorry

File: assessments/2022-12-08-control-constructs/code.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  for (let i = 0; i < s.length; i++) {
+    console.log(i)
+    if (s[i] === 'x') {
+      return true
+    }
+    return false
+  }
+};// THIS SHOULD BE LOOPING WHY ISNT IT AAAAAAAA
+// im going fucking insane why isnt this wokring
+// its gonna be some really mundane shit like a slight typo that i missed or something and im gonna be fucking LIVID
```
Comment (gigamonkey) [inline assessments/2022-12-08-control-constructs/code.js:33]: I'm afraid to report that you are correct. Look at line 29 and think about where it is relative to the body of the loop. Other than that though this is correct. (Sorry man, this is how programming is sometimes. I have spent hours on bugs and then been like, "oh. shit. duh.")

File: assessments/2022-12-08-control-constructs/code.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    if (s[i] === 'x') {
+      xs++
+    }
+  }
+  return xs
+};
+
+const timesTable = (n) => {
+  return
+}; // complicated :(
```
Comment (gigamonkey) [inline assessments/2022-12-08-control-constructs/code.js:21]: It is a bit. Best bet, check the posted answer and make sure you understand it.

**Review by gigamonkey — APPROVED** [review]
Despite some misery along the way, you did quite well. And thanks for remembering to request review, even if it was a bit late. Under the new Github grading regime that will actually help you out, especially if you go ahead and merge this.

**Comment by AWalnut1** [comment]
changed the mistake and removed any stray console.logs

**Pull request merged**